.devcontainer/docs/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM node:18-alpine
+
+# Install git
+RUN apk add --no-cache git
+
+# Install docsify-cli
+RUN npm install -g docsify-cli
+# Install prettier and eslint for prettifying and linting
+RUN npm install -g prettier eslint
```

```suggestion
RUN npm install -g docsify-cli prettier eslint
```

Smaller images are better.